utils/torch_solver.py

```diff
@@ -29,9 +38,10 @@ def run(self, callback):
 
         # Setup the trainer
         trainer = Trainer(
-            max_epochs=-1, callbacks=[BenchoptCallback(callback)]
+            max_epochs=-1, callbacks=[BenchoptCallback(callback)],
+            accelerator="auto", devices=1
```

Comment:
Here as well we should have a todo to say that we need a better way to set the number of gpus